package_manifest.yaml

```diff
@@ -126,6 +126,7 @@ foreman_packages:
     nodejs-webpack-stats-plugin: {}
     nodejs-webpack: {}
     nodejs-zxcvbn: {}
+    pcp-mmvstatsd: {}
```

Comment:
The build fails because this doesn't match the package name.

Reply:
This is now fixed.